src/pudl/package_data/notebooks/how_to_use_pudl_out.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>Might want to highlight the <code>demand_hourly_pa_ferc714()</code>  table here since it's the one we've actually worked to clean up, and is likely the most valuable table in this dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>great!</p>